api/protobuf/core/api_container_service.proto

```diff
@@ -68,6 +68,11 @@ message Port {
   string maybe_wait_timeout = 4;
 }
 
+enum ServiceStatus {
+  STOPPED = 0;
+  RUNNING = 1;
```

Comment:
Do we need `REMOVED` as well here? My memory is a bit fuzzy here, but since we're not removing the container but just stopping it, do we keep the service in memory at the APIC level and should we return it as `REMOVED` here? Probably not, but just double checking

Reply:
We don't keep removed services at that level.